core/src/impl/Kokkos_Core.cpp

```diff
@@ -683,6 +683,8 @@ void pre_initialize_internal(const Kokkos::InitializationSettings& settings) {
 #elif defined(KOKKOS_ARCH_AMD_ZEN3)
   declare_configuration_metadata("architecture", "CPU architecture",
                                  "AMD_ZEN3");
+#elif defined(KOKKOS_ARCH_RISC64_rv64imafdcv)
```

Comment:
You would need to add a `#cmakedefine` in `cmake/KokkosCore_config.h.in` for this macro to be defined.

Reply:
Ok, I added the define